docs/testing/cart-checkout/readme.md

```diff
@@ -0,0 +1,191 @@
+
+## What are we testing?
+
+The goal is to test the new Cart and Checkout blocks, they should be replacing
+the Cart and Checkout shortcodes.
+
+### Cart Block
+
+![](https://i.imgur.com/mcbXgqV.png)
+
+#### Features to test:
+
+The Cart block has a user story with all [expected features here.](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/1289)
+<!-- debating whether we should be linking to the epic, or copying the epic
+content here, in both cases, the epic should be cleaned and features that are
+not valid for this MVP should be deleted to avoid confusion. -->
+You are also encouraged to compare behaviors with the regular Cart block.
+
+
+### Checkout Block
+
+![](https://i.imgur.com/9KhYK2L.png)
+
+#### Features to test:
+
+The Checkout block has a user story with all [expected features here.](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/1294)
+<!-- same comment as above -->
+You are also encouraged to compare behaviors with the regular Checkout block.
+
+## Testing Checklist
+
+### Editor experience
+
+- [ ] You should be able to add, and only one Cart or Checkout block to a page.
+- [ ] The preview in the inserter should show a skeleton of how the block structure should look like.
+- [ ] When inserting any of the blocks, it should have some data already in it.
+- [ ] You should not be able to interact directly with the block (except for some sections).
+- [ ] You should be able to see block settings on the sidebar when it is focused.
+- [ ] Proceed to Checkout and Back to cart block settings should present you with a list of your website pages.
+
+### Shipping
+
+#### Setup
+
+
+With shipping zones available:
+
+- [ ] You should be able to see preview rates (that are not your actual rates) in the editor.
+- [ ] You should be able to see your actual rates on the frontend.
+- [ ] Selecting a shipping rate should update the totals.
+- [ ] Changing the address in Cart block should update the rates.
+- [ ] Try entering an address that does not have rates for, you should:
+  - [ ] See an error saying "No options were found".
+  - [ ] See the default shipping option if you have it setup.
+- [ ] The countries in the shipping rates form should reflect the countries you have in WooCommerce -> Settings -> General -> Shipping location(s).
+- [ ] If your cart has only digital products, the Cart and Checkout blocks should act like shipping is disabled.
+
+If you don't have any shipping zones set up and/or shipping is disabled:
+
+- [ ] You should only see the billing form in both editor and frontend for the Checkout Block.
+- [ ] The shipping options step should not be visible.
+- [ ] The shipping cost should not be visible in the sidebar.
+
+If you don't have any shipping zones set up but **shipping is enabled**:
+
+- [ ] In the editor, Checkout Block will show you a placeholder promoting you to set up shipping zones.
```

Comment:
I feel like I went way to specific here, if we continue like this, this checklist will take forever to prepare.

Reply:
The checklist will take a while to prepare, yes. But then it remains for all future testing of the blocks whenever we make changes.